### PR TITLE
Brinda l210.resources china

### PR DESCRIPTION
**Pull request by brinday**

### CODE REVIEW

File: R/zchunk_L210.Resources_china.R [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    L210.RenewRsrc_CHINA <- L210.RenewRsrc %>%
+      filter(region == "China",
+             renewresource %in% gcamchina.PROVINCE_RENEWABLE_RESOURCES) %>%
+      write_to_all_provinces(LEVEL2_DATA_NAMES[["RenewRsrc"]]) %>%
+      # Remove geothermal from provinces that don't have it
+      anti_join(no_geo_provinces_resource, by = c("region", "renewresource")) %>%
+      mutate(market = if_else(renewresource != "onshore wind resource", "China", region))
+
+    # L210.UnlimitRsrc_CHINA: unlimited resource info in the provinces
+    # TODO: If needed, add in capacity factor (from old data system, seems to be 0.3 for solar, 0 for limestone)
```
Comment (bpbond) [inline R/zchunk_L210.Resources_china.R]: Capture this in a GitHub issue? Up to you.

File: R/zchunk_L210.Resources_china.R [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    no_geo_provinces <- gcamchina.PROVINCES[ gcamchina.PROVINCES %in% L1231.out_EJ_province_elec_F_tech$province[ L1231.out_EJ_province_elec_F_tech$fuel == "geothermal" &
+                                                                                                                     L1231.out_EJ_province_elec_F_tech$X2010 == 0 ] ]
+    no_geo_provinces_resource <- tibble( region = no_geo_provinces, renewresource = "geothermal")
+
+    # L210.RenewRsrc_CHINA: renewable resource info in the provinces
+    L210.RenewRsrc_CHINA <- L210.RenewRsrc %>%
+      filter(region == "China",
+             renewresource %in% gcamchina.PROVINCE_RENEWABLE_RESOURCES) %>%
+      write_to_all_provinces(LEVEL2_DATA_NAMES[["RenewRsrc"]]) %>%
+      # Remove geothermal from provinces that don't have it
```
Comment (bpbond) [inline R/zchunk_L210.Resources_china.R]: Clear, succinct comment 👍

File: R/zchunk_L210.Resources_china.R [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+      write_to_all_provinces(LEVEL2_DATA_NAMES[["UnlimitRsrcPrice"]])
+
+    L210.UnlimitRsrcPrice_limestone_CHINA <- L210.UnlimitRsrcPrice_CHINA %>%
+      filter(unlimited.resource == "limestone",
+             region %in% cement_provinces)
+
+    L210.UnlimitRsrcPrice_CHINA <- L210.UnlimitRsrcPrice_CHINA %>%
+      filter(unlimited.resource != "limestone")
+
+    # TODO: L210.SmthRenewRsrcTechChange_CHINA (look at old data system code chunk) can be added back in when we have distributed_solar added by province
```
Comment (bpbond) [inline R/zchunk_L210.Resources_china.R]: Same; feel free to make a `gds-china` issue label @sypnl

Comment (sypnl) [reply R/zchunk_L210.Resources_china.R]: opened a new issue on provincial vs. national resource curves (todos marked in this chunk). Used the gcam-china issue label created last year.

**Pull request merged**